packages/terra-dropdown-button/docs/SplitButton.md

```diff
@@ -0,0 +1,15 @@
+# Split Button
+
```

Comment:
Is this two buttons or one? If I understood the design this is technically "one" button for user interactions so describing it as two is confusing

Reply:
@jenniferwurth has provided some wording for this on the GitHub issue: https://github.com/cerner/terra-core/issues/2216#issuecomment-501327457